kafka/signed_blinded_token_issuer_handler.go

```diff
@@ -44,7 +44,7 @@ func SignedBlindedTokenIssuerHandler(
 			continue
 		}
 
-		issuer, appErr := server.GetLatestIssuer(request.Issuer_type, int(request.Issuer_cohort))
+		issuer, appErr := server.GetLatestIssuer(request.Issuer_type, int16(request.Issuer_cohort))
```

Comment:
I think this could produce a potential overflow, although it wont error

Reply:
https://github.com/brave-intl/challenge-bypass-server/pull/145/commits/2095b05667b6175637a40aead6ea3dbde57e4a01